mods/git.py

```diff
@@ -109,7 +114,11 @@ def mark_as_pending_apply(self, series, data={}):
         r.save()
 
     def on_tags_update(self, event, series, **params):
-        if series.is_complete:
+        config = self.get_project_config(series.project)
+        update_series_on_tags = config["update_series_on_tags"] \
+                                if "update_series_on_tags" in config else True
```

Comment:
This can be simplified as `update_series_on_tags = config.get("update_series_on_tags", True)`?

Reply:
good idea!